src/registrar/admin.py

```diff
@@ -1114,7 +1114,7 @@ class SeniorOfficialAdmin(ListHeaderAdmin):
     # NOTE: these are just placeholders.  Not part of ACs (haven't been defined yet).  Update in future tickets.
```

Comment:
We can delete this comment.  It's a remnant from one of my old tickets and it looks like people are happy with these settings.